### ISSUE TITLE
[Card] Accessibility violation for no aria-label in card header tooltip.

### ISSUE THREAD

**Issue by kalyanixraut** [carbon-design-system/carbon-addons-iot-react#3395]
### What package is this for?

- [x] React
- [ ] Angular

### Describe the bug
Card is used in Maximo Graphite and accessibility violation is thrown for the card header tooltip.
<img width="804" alt="image" src="https://user-images.githubusercontent.com/22093326/159880591-feb369e2-ac02-4fc1-9c6e-f2415c60041a.png">

### To Reproduce

1. Go to storybook https://next.carbon-addons-iot-react.com//?path=/story/1-watson-iot-card-card--with-ellipsed-title-tooltip-external-tooltip
2. Check the accessibility tab

### Expected behavior

No accessibility violation for card header tooltip.

### Environment/versions:

- OS: [e.g. MacOS, Windows]
- Browser: [e.g. chrome, safari]
- `carbon-addons-iot-react` version: 2.149.0-next.10
